.circleci/config.yml

```diff
@@ -17,8 +18,24 @@ jobs:
           command: |
             set -x
             aws cloudformation deploy --template-file cloudformation/sample_stack.yml --stack-name sampleStack
+  execute_ansible:
+    executor: ansible-playbook/default
+    circleci_ip_ranges: true
```

Comment:
## CAUTION!
- 2023/12/16現在、freeプランだとこの機能は使えない
refs: https://circleci.com/docs/ja/ip-ranges/

### freeプランで指定した場合
- 以下のようなエラーメッセージが表示されてジョブが失敗する
> This job has been blocked because the IP ranges feature is not available on your plan. Please upgrade to continue building.

refs: https://app.circleci.com/pipelines/github/yuta-ushijima/raisetech-live15-demo-20231216/10/workflows/ef7ce0e1-e274-4e68-af71-22f15ae097fa/jobs/13